loader/vulkan-1.def

```diff
@@ -1,9 +1,9 @@
 
 ;;;; Begin Copyright Notice ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;
-; Copyright (c) 2015-2017, 2019 The Khronos Group Inc.
-; Copyright (c) 2015-2017, 2019 Valve Corporation
-; Copyright (c) 2015-2017, 2019 LunarG, Inc.
+; Copyright (c) 2015-2021 The Khronos Group Inc.
```

Comment:
Do we generate this file?  If not, maybe we should make an issue in GitHub to generate it based on the LOADER_EXPORT define and then we don't have to worry about missing something.

Reply:
Currently this file is not generated. This is definitely something I would like to auto-generate in the future, just hasn't been high priority. Issue #504 documented the ability for these files to be generated, so no issue needs to be made.